docs/demo.md

```diff
@@ -0,0 +1,183 @@
+# Image DePHI Demo
+
+This walkthrough will guide you through using the Image DePHI program.
+
+## Getting the demo data
+
+In order to get the demo data, you will need to have installed Image DePHI and run the following command:
```

Comment:
Elsewhere we use "ImageDePHI" as a single word.  I think I prefer that to having the space between Image and DePHI.